policies/branch-policy.md

```diff
@@ -0,0 +1,100 @@
+Branch Policy
+=============
+
+The openssl repository contains the following maintained branches:
+
+## The master branch
```

Comment:
Nit: `##` should be `###`, because you have a level two title further down (underlined with `--------...`)